python/foglamp/services/common/microservice.py

```diff
@@ -50,15 +50,25 @@ class FoglampMicroservice(FoglampProcess):
     _protocol = "http"
     """ communication protocol """
 
-    def __init__(self, default_config):
+    def __init__(self):
         super().__init__()
         try:
+            # Configuration handled through the Configuration Manager
+            default_config = {
+                'local_services': {
+                    'description': 'Restrict microservice to localhost',
+                    'type': 'string',
+                    'default': '127.0.0.1',
+                }
+            }
+
             loop = asyncio.get_event_loop()
 
-            # ----- Ref: FOGL-1155. We need to fetch host from configuration of plugin
-            category = self._name
+            # ----- Ref: FOGL-1155/1439.
```

Comment:
we can change the base to 1432 or keep comment  of 1432

Reply:
Let us keep it independent of 1432.

Reply:
please remove this and change the base to 1432.